test/test_custom_ops.py

```diff
@@ -2550,6 +2551,56 @@ def sin_(x):
         sin_(x)
         self.assertEqual(x, expected)
 
+    @skipIfTorchDynamo("Expected to fail due to no FakeTensor support; not a bug")
+    def test_subclass_accessor_view(self):
+        class MyTwoTensor(TwoTensor):
+            @classmethod
+            def __torch_dispatch__(cls, func, types, args, kwargs):
+                if func is torch.ops._torch_testing._two_tensor_accessor.default:
+                    self.assertIsInstance(args[0], MyTwoTensor)
+                    res = args[0].a
+                    # Always return a fresh Tensor!
+                    return res.view_as(res)
```

Comment:
No action necessary, but now that custom ops have official support for views, I wonder if there's things we can do to make the inference mode handling smoother. 

With this PR, I guess wrapper subclasses still need special handling for inference mode for views (where ordinarily inference-ness being preserved via this line, but can be broken if the outer tensor must be rewrapped in a new subclass)